docs/README.md

```diff
@@ -1,19 +1,21 @@
-# webots-doc
+# Webots Documentation
 
-This repository contains the documentation of the Webots software.
+This repository contains the documentation for the Webots open source simulation software.
 
-The master branch of this repository is in production on the Cyberbotics website:
+The master branch of this repository is in production on the Cyberbotics website, which you can access by visiting the following links:
 
 - https://www.cyberbotics.com/doc/guide/index
 - https://www.cyberbotics.com/doc/reference/index
 - https://www.cyberbotics.com/doc/blog/index
 - https://www.cyberbotics.com/doc/automobile/index
 
-You are very welcome to contribute to make this documentation better!
-In order to proceed, simply fork this repository, make your modifications and
-open a pull request that we will review and merge.
+You are very welcome to contribute to make this documentation better.<p>
+In order to proceed, follow these steps:
+1. Simply fork this repository <p>
+2. Make your modifications &<p>
+3. Open a pull request that we will review and merge.<p>
```

Comment:
We should avoid `<p>` in markdown as it is usually not needed.

Reply:
Should <br/> be used instead of <p> then?

Reply:
If you want to put some vertical space, an empty line should suffice. But it may not be necessary...

Reply:
Also we should avoid & signs which are not very literal, prefer "and".

Reply:
Okay, I'll do that and will commit these changes.

Reply:
I have tried to suffice the requirements.